invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunitiesList.js

```diff
@@ -54,6 +62,11 @@ export class RecordCommunitiesList extends Component {
               <Item.Header as={Header}>
                 <Header as="a" href={community.links.self_html} size="small">
                   {community.metadata.title}
+                  {community.parent && (
+                    <HeaderSubheader>
+                      Part of {community.parent.metadata.title}
+                    </HeaderSubheader>
+                  )}
                 </Header>
                 {community.id === branded && community?.theme && (
```

Comment:
```suggestion
                {community.is_verified && (
```
Should this be changed  so all verified communities have the verified icon, as we're already doing in other places